---
layout: page
title: CV
permalink: /cv/
---

# Skills
## Management: 
Conceptualisation | Team coordination | Experimental design | Project planning | Scheduling | Resources management | Fund management | Data archiving | CRM | Agile
## Communication: 
Presentation | Teaching | Mentoring | Collaborating | Publishing significant contributions to the scientific community, with 16 peer-reviewed publications (full [publication](https://djleite.github.io/publications) list) in journals with impact factors ranging from 3 to 15, collectively cited over 700 times. 
## Computational: 
Bioinformatics | Python | R | Bash | HPC | Linux/Mac/Win OS | Command-line | Tool development | Genome assembly | Genome annotation | Single-cell sequencing | non-coding DNA/RNA annotation |  Transcriptomics | Phylogenetics | SNP/Variant Calling | Visualisation | Statistics | Microsoft Office | Data archiving | GitHub
## Laboratory techniques: 
DNA/RNA extraction | NGS library preparation | PCR | Cloning | in-situ hybridisation | Molecular biology | Immunohistochemistry | Microscopy (bright-field/confocal/SEM/TEM) | Invertebrate husbandry | Species identification


# EXPERIENCE

## Senior Bioinformatician and Business Developer - Pragmatic Genomics (2024 May - Oct)
### Responsibilities:
- Developed genomics SaaS tools for cloud-based analyses
- Conceptualised ideas, reviewed and tested using agile practices and GitHub version control
- Established and managed high and low-touch sales funnels
- Improved CRM (Hubspot), customer engagement, automation (Make), and feedback integration
- Optimised website (HTML/CSS), improved design, SEO (Lighthouse), online resources (blogs, tutorials, functionality information), and branding
### Achievements:
- Improved front/back-end tool functionality, UX, security, and ensured biological and scientific rigour
- Increased B2B and B2C sales over £50k pa via improved sales funnels and CRM methods
- Overhauled website content, performance, and resources to attract and support customers

## Postdoctoral Researcher – Durham University (2020 – 2024)

### Responsibilities:
- Co-led team coordination, international collaborations, resource and fund management
- Conceived experimental design, technical analyses and scientific interpretations 
- Managed data collections, archiving, task designation, communications and publications
- Mentored and supervised Postdocs/PhDs/undergraduates, teaching computational analyses
### Achievements:
- Discovered and published new insights of spider development at single-cell resolution 
- Assembling and annotating four new spider genomes using NGS long-read (PacBio HiFi/Iso-Seq) and short-read (Illumina DNA/RNA-Seq and Arima HiC) technologies
- Developed novel bioinformatic approaches for studying genome synteny and evolution
### Projects:
- Single-cell RNA-Seq (SPLiTseq) transcriptomics analyses of spider development 
- Systematic identification of duplicate genes in chelicerate whole genome duplications

## Associate Lecturer – Oxford Brookes University (2021 – 2023)
### Responsibilities:
- Design course material and examinations, in coordination with co-lecturers
- Taught theoretical and bioinformatic practical sessions and mentoring undergraduate students
### Achievements:
- Received positive feedback from students and co-lecturers, evidenced by high exam scores

## Postdoctoral Researcher – University College London (2018 – 2020)
### Responsibilities:
- Co-led an international trip to collect live animals, transport, care, complied to law and regulations
- Conceived of experimental design, technical analyses and scientific interpretations 
- Managed data collections, archiving, collaborations and publications
- Mentored PhDs and In2Science outreach, teaching wet-lab and computational analyses
### Achievements:
- Sequenced, assembled and published the first polyclad flatworm genome
- Learnt single-cell wet-lab and bioinformatic analyses
### Projects:
- Genome assembly (PacBio CCS) and annotation of a polyclad flatworm
- Single-cell analysis of early development of spiral cleavage and larva in lophotrochozoans
- Phylogenetic relationships of major animal clades

## Postdoctoral Researcher – Oxford Brookes University (2017 – 2018)
### Responsibilities:
- Managed data collection, analyses, and archiving, and publications  
- Conceived experimental design, technical analyses and scientific interpretations 
- Mentored and supervised PhDs/undergraduates, teaching computational and wet-lab analyses
### Achievements:
- Established international collaborations, acquiring resources that expanded scientific perspective
- First characterisation of all homeobox genes in spiders and support for whole genome duplication
### Projects:
- Investigated the homeobox gene repertoires in chelicerate whole genome duplications
- Genome assembly (SPAdes) and variant calling (GATK) between strains of baculoviridae


# Education
## Ph.D. – Oxford Brookes University (2013 – 2017)
Thesis “Duplication and divergence of homeobox genes and microRNAs in Chelicerata”.
Published three first-author papers and received accolades for presentations and posters.
## BSc– Oxford Brookes University (2010 – 2013)
1st Class Honours. Evolutionary | Developmental | Molecular | Cellular | Plant | Conservation


# Other
## Journal Reviewer: 
BMC Genomics | BMC Biology | Proc. of the Royal Society London B | Nucleic Acids Research | Development Genes and Evolution | JEZ Part B: Molecular and Developmental Evolution
## Key conference presentations: 
Keynote at DZG, Kassel, Germany (2023)
Invited speaker at Durham, UK (2022)
Speaker at Darwin Tree of Life, UK (2021)
Speaker at SpiderWeb, Göttingen, Germany (2017)
Speaker at SpiderWeb, Jena, Germany (2014)
## Teaching and outreach:
Associate lecturer (2021 – 2023)
Supervisor of undergraduates, masters, and Ph.D. students (2013 – 2024)
Outreach with In2Science (2019) and primary schools (2014 – 2017)
## Gatsby Plant Summer School (2011):
Experience and training in ideas and solutions in plant science.