---
title: Chelicerate whole genome duplications
date: 2025-05-08
tags: [featured]
image: /images/spider-wgd-full.webp
excerpt: Impact of genome evolution in chelicerate lineages with whole genome duplication
---

## Background

Whole genome duplication (WGD) contributes large additions of genetic material that impacts genome evolution. The evolution of additional coding and non-coding elements can alter the regulation of developmental processes, resulting in phenotypic change and diversification. Analyses of genomic and transcriptomic data has revealed that the arthropod subphylum of Chelicerata contains multiple WGDs. This includes a single WGD in the ancestor of arachnopulmonates (spiders, scorpions, and their relatives), and independently three WGDs in the lineage of horseshoe crabs (fig 1).

![Chelicerate phylogeny](/images/chelicerate-phy.webp)
<p style="text-align:left; font-size: 0.85rem; line-height: 1.2;"><b>Figure 1: The arthropod phylogeny.</b> Representatives of chelicerates, including spiders (<i>P. tepidariorum</i>, <i>S. mimosarum</i>, and <i>A. geniculata</i>) and scorpions (<i>C. sculpturatus</i> and <i>M. martensii</i>), a tick (<i>I. scapularis</i>), a mite (<i>T. urticae</i>), and a horseshoe crab (<i>L. polyphemus</i>), as well as representatives of Myriapoda (<i>S. maritima</i>), Crustacea (<i>D. pulex</i>), and Insecta (<i>D. melanogaster</i> and <i>T. castaneum</i>).</p>

## Genomes of spiders and scorpions
Our analysis of some of the first spider and scorpion genomes (<a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-017-0399-x" target="_blank" rel="noopener noreferrer">Schwager <i>et al</i> 2017</a>) revealed evidence of WGD. The evidence included elevated duplication levels of BUSCO genes, duplication of many developmental genes (e.g. [homeobox genes](/spider-wgd#homeobox-genes) and [Sox genes](/spider-wgd#sox-genes)), conservation of synteny between genomic scaffolds and gene clusters, and phylogenetics signatures from gene tree topologies.

## Homeobox genes
One of the hallmarks of WGD in animal lineages are the presence of duplcate Hox clusters. In the <i>P. tepidariorum</i> and <i>C. sculpturatus</i> genomes, we identified almost intact duplicate Hox clusters (<a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-017-0399-x" target="_blank" rel="noopener noreferrer">Schwager <i>et al</i> 2017</a>).

Phylogenetic surveys of genomic and transcriptomic resources across arthropods systematically identified homeobox gene duplication (<a href="https://academic.oup.com/mbe/article/35/9/2240/5040134" target="_blank" rel="noopener noreferrer">Leite  <i>et al</i> 2018</a>). This analysis captured a clear signal that approximately 50% of homeobox gene families were duplicated in arachnopulmonates, with conservation of homeobox family duplication across arachnopulmonate species.

![Arthropod homeobox genes](/images/spider-homeobox.webp)
<p style="text-align:left; font-size: 0.85rem; line-height: 1.2;"><b>Figure 2: Comparison of homeobox repertoires in arthropods reveals pervasive duplication in arachnopulmonates.</b> The copy number of homeobox families is generally greater in arachnopulmonates compared with other arthropods.</p>

Surveys of homeobox genes duplication have been extended as genomic resources have improved in contiguity and diversity (<a href="https://academic.oup.com/mbe/article/40/12/msad239/7341927" target="_blank" rel="noopener noreferrer">Aase-Remedios <i>et al</i> 2023</a>). Given the improvement of contiguity we have been able to contextualise the syntenic relationships of homeobox clusters, and how their rearrangements between species coincide with chromosomal rearrangements (fig 3).

![Spider WGD synteny](/images/spider-wgd-full.webp)
<p style="text-align:left; font-size: 0.85rem; line-height: 1.2;"><b>Figure 3: Syntenic relationships between spider genomes.</b> Homeobox cluster rearrangement coincide which genomic rearrangements.</p>

## Sox genes
Sox genes, like homeobox genes, are important transcription factors that instruct many aspects of development (<a href="https://elifesciences.org/articles/37567" target="_blank" rel="noopener noreferrer">Paese <i>et al</i> 2018a</a>; <a href="https://bmcecolevol.biomedcentral.com/articles/10.1186/s12862-018-1337-4" target="_blank" rel="noopener noreferrer">Paese <i>et al</i> 2018b</a>). We phylogenetically identified Sox genes in arthropods and similarly found spiders to contain more Sox genes compared with other arthopods surveyed.

![Spider Sox genes](/images/spider-sox.webp)
<p style="text-align:left; font-size: 0.85rem; line-height: 1.2;"><b>Figure 4: Sox gene copy numbers in arthopods.</b> Sox genes are duplicated in spiders more so than other arthropods surveyed.</p>

## microRNAs
Signatures of WGD are usually assessed in terms of coding gene duplication. However, it is also important to understand how non-coding elements are duplicated and retained. MicroRNAs (miRNAs) are short ~22 nt double stranded RNA molecules that form complexes with proteins, that collectively bind and suppress mRNA translation.

Using small RNA-Seq and bioinformatic pipelines we were able to identify miRNAs in the genome of the spider <i>P. tepidariourm</i> (<a href="https://academic.oup.com/gbe/article/8/7/2133/2466064" target="_blank" rel="noopener noreferrer">Leite <i>et al</i> 2016</a>). This revealed that many conserved metazoan miRNA families were present in duplicate. Surveying across arthropods, it was evident that arachnopulmonates and horseshoe crabs exhibited greater miRNA duplication compared to other arthopods (fig 5). This highlighted the importance of considering both coding and non-coding genomic elements when investigating WGD.

![microRNAs in arthopods](/images/spider-mirna.webp)
<p style="text-align:left; font-size: 0.85rem; line-height: 1.2;"><b>Figure 5: miRNA copy numbers in arthopods.</b> miRNAs are duplicated in spiders, scorpions and horseshoe crabs more so than other arthropods surveyed.</p>

## Impact
To understand the impact of WGD and the downstream consequences, we used the spider <i>P. tepidariourm</i> to look at the embryonic expression. Using techniques like immunohistochemistry (ISH) and microscopy, we were able to see where and when genes were expressed. Coupled with gene disruption techniques like RNAi we could also see the potential roles of each gene.

Hox genes showed remarkable changes in their anterior-posterior and temporal expression, highlighting the changes in body plan regulation (<a href="https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-017-0399-x" target="_blank" rel="noopener noreferrer">Schwager <i>et al</i> 2017</a>). These changes could be seen in [single-cell RNA-Seq dataset](/spider-single-cell) (<a href="https://evodevojournal.biomedcentral.com/articles/10.1186/s13227-024-00224-4" target="_blank" rel="noopener noreferrer">Leite<i>et al</i> 2024</a>), where duplicates were expressed in different cell clusters, suggesting that duplicates have undergone sub- and neo- functionalisation.

Additionally, Sox gene duplication and evolution showed interesting divergences in [posterior segmentation regulation](/spider-segmentation), both with the spider and compared to other arthopods (<a href="https://elifesciences.org/articles/37567" target="_blank" rel="noopener noreferrer">Paese <i>et al</i> 2018a</a>; <a href="https://bmcecolevol.biomedcentral.com/articles/10.1186/s12862-018-1337-4" target="_blank" rel="noopener noreferrer">Paese <i>et al</i> 2018b</a>).

